controllers/heat_controller.go

```diff
@@ -713,12 +715,7 @@ func (r *HeatReconciler) generateServiceConfigMaps(
 			Labels:        cmLabels,
 		},
 	}
-	err = configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
-	if err != nil {
-		return nil
-	}
-
-	return nil
+	return configmap.EnsureConfigMaps(ctx, h, instance, cms, envVars)
```

Comment:
Ah nice catch on the unnecessary error handling there too.

Reply:
Actually the current implementation is wrong because it completely ignores the failure.